ampd/src/solana/msg_verifier.rs

```diff
@@ -177,16 +180,16 @@ mod tests {
         // tests/solana_tx.json file and use it as test fixture. See the "logMessages" field
         // on it.
 
-        // println!(
-        //     "------> {}",
-        //     get_tx_log_message(
-        //         source_address.clone(),
-        //         destination_chain.clone().into_bytes(),
-        //         destination_address.clone().into_bytes(),
-        //         payload,
-        //         payload_hash
-        //     )
-        // );
+        println!(
+            "------> {}",
+            get_tx_log_message(
+                source_address.clone(),
+                destination_chain.clone().into_bytes(),
+                destination_address.clone().into_bytes(),
+                payload,
+                payload_hash
+            )
+        );
```

Comment:
This is just for generating the correct log line for the transaction, should be commented once we are done with testing.

Reply:
I was thinking on this and i consider it a good idea. But being this not our codebase, i was being conservative about adding more dependencies to the project. It's a dev dependency though.